src/components/molecules/DragNDropList/DragNDropList.tsx

```diff
@@ -15,8 +15,8 @@ interface ItemComponentProps {
 }
 
 type DragNDropListProps = {
-  items: TestSuiteStep[];
-  setItems: (steps: TestSuiteStep[]) => void;
```

Comment:
Not important - probably it should be `any[]`, as it's not specific for test suite steps. Additionally, `value`/`onChange` would match better than `items`/`setItems`.